### ISSUE TITLE
Getting E208 when creating a symlink with file module

### ISSUE THREAD

**Issue by Benjamin-cholet** [ansible/ansible-lint#994]
##### Summary
Since v4.3.0 and #949 , an E208 (file permissions not mentioned) error is raised when creating a link with the ansible **file** module.
In my opinion, it makes no sense to require permissions when creating a symlink.

##### Step to reproduce

ansible-lint version 4.3.1
ansible version 2.9.11
(both installed with pip)
OS: Centos 7.8

Sample playbook:

```
# test.yml
---
- name: test
  hosts: localhost
  tasks:
    - name: "create a file"
      file:
        path: "foo"
        state: touch
        mode: 0755
    - name: "create a symlink"
      file:
        path: "bar"
        state: link
        src: "foo"
```

```
$ ansible-lint test.yml
[208] File permissions not mentioned
test.yml: 13
Task/Handler: create symlink
```

##### Desired Behaviour

No E208 returned when state == "link"

##### Actual Behaviour

An E208 is returned when state == "link"

**Comment by greg-hellings** [comment]
@Benjamin-cholet Can you test this against the current git master? I'm looking in the Rule file, and it appears that it accounts for your situation both in the Rule and in the test for it.

**Comment by ssbarnea** [comment]
Fixed on master

**Comment by Benjamin-cholet** [comment]
Apologies I missed that.
Thank you
